frontend/src/metabase/core/components/Toggle/Toggle.tsx

```diff
@@ -27,6 +28,26 @@ const Toggle = forwardRef(function Toggle(
     [onChange],
   );
 
+  if (label) {
```

Comment:
It's worth noting that we have `<FormToggle />` which is normally used when we use Formik forms, and is more sophisticated. But it renders differently 
<img width="645" alt="image" src="https://user-images.githubusercontent.com/1937582/229906371-dc791ae4-e7f9-4139-8f01-4f1f46e0785d.png">
 
So, it makes sense to me at this point to support `label` within the `Toggle` itself, which is a lighter and simpler version of the `FormToggle`.
<img width="320" alt="image" src="https://user-images.githubusercontent.com/1937582/229906660-b5d11ee4-8078-4574-b047-209641d4c0c6.png">

Reply:
If you need a `Toggle` with a label that is not within the form, you could do 

```
<FormField title={t`Auto-apply filters`} ...>
  <Toggle ... />
</FormField>
```

I don't think we should add another way of adding a label to a toggle/checkbox, wdyt?

Reply:
Furthermore, I don't think we should be including `FormField` within the component itself, so I'd rather use the existing approach and use `FormField` + `Toggle` on the consumer side

Reply:
Using `FormField` sounds good!

But, I saw that we have different kinds of styling labels too (see screenshots in the original comment). A more straightforward way of working that out is to add a styled file like `ChartSettingsWidget.tsx` here.
https://github.com/metabase/metabase/blob/6e03c5a663805852793b0e9da448259670ff12f4/frontend/src/metabase/visualizations/components/ChartSettingsWidget.tsx#L62-L66

<img width="323" alt="image" src="https://user-images.githubusercontent.com/1937582/230068922-4d039421-ade2-49c7-8d3c-464758ec3124.png">

The second approach could be to add a `labelVariant` (or any other better name) prop to `FormField`, so that we could standardize this design (which I think should even replace the ones we used in admin settings.

I'm in favor of adding a prop to `FormField` for different labels, what do you think?